relay-server/src/http.rs

```diff
@@ -331,3 +348,27 @@ impl Response {
         }
     }
 }
+
+fn reqwest_body_from_read<R>(mut reader: R) -> reqwest::Body
```

Comment:
_nit_: Please move this function before its first use and add a doc comment.

Reply:
There is a doc comment on it (weirdly I got an email for your comment that had the old diff), but yeah I moved the function (we need a linter for that...)